volumes/miovision/README.md

```diff
@@ -229,7 +229,7 @@ An example of `miovision_api.movement_map`:
 ![TMC movements](img/intersection_tmc.png)
 
 **Blue line represents `leg_old` whereas yellow arrows represent `movement_uid`**
```

Comment:
This image should be fixed I think. Or you could extend the blue line or reverse the yellow arrows. The blue line half represents `leg_new`, it does not represent `leg_old`, that should be the originating leg.

Reply:
Do you know where Joven kept the (PPT? Illustrator?) file that makes this picture?

Reply:
😭 
that would have been a good idea to ensure future maintainability.
Don't worry if the graphic sounds like a lot of effort, I would focus on clarifying the error in the text and maybe emphasize that the graphic is a... WIP

Reply:
Did some Photoshopping, and revised the diagrams and surrounding text.